### PR TITLE
Explicitly log that setup.sh is missing when using custom setup scripts

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/conf/setup/custom.sh [hunk 1]
```diff
@@ -36,5 +36,10 @@ for dir in $(ls -1 /customer-scripts | grep _ | grep -oE '^.*?_' | sed 's/_//g'
   for file in $(ls -1 "/customer-scripts/${dir}_"* | grep -oE '_.*' | sed 's/_//g'); do
     cp "/customer-scripts/${dir}_${file}" "${target}/${file}"
   done
+
+  [[ ! -f setup.sh ]] \
+    && echo "You're missing setup.sh script in custom scripts directory: '${dir}'" \
+    && continue
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/conf/setup/custom.sh:42]: I think `if then` would be better to use here than oneliner

Comment (pmalek-sumo) [reply deploy/helm/sumologic/conf/setup/custom.sh]: Please recheck

**Review by sumo-drosiek — APPROVED** [review]
:+1:

**Comment by pmalek-sumo** [comment]
@perk-sumo @sumo-drosiek Please +1 I had to make commits pretty 🙄

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**